go.mod

```diff
@@ -1,5 +1,9 @@
 module kubevirt.io/hostpath-provisioner-operator
 
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
I would recommend this way of specifying the patch version over the current
https://github.com/kubevirt/kubevirt/pull/12130
Otherwise this could be a little harder to ensure the project is built properly on different platforms

Reply:
Fixed